content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Configuring Virtualization and Apps in TrueNAS SCALE"
+weight: 50
+tags:
+- scalevm
+- scaleapps
+- scalenetwork
+- scalevlan
+- scalelagg
+- scaledatashare
+---
+
+{{< toc >}}
+
+Configuring TrueNAS SCALE to work with virtualized features, such as VMs and Apps, is a part of the setup process that when optimized takes advantage of the network storage capabilities that SCALE offers. 
+
+## Before You Begin
+
+This article assumes that you have the latest release version of TrueNAS SCALE installed on your system. The following steps comprise a list of configuration prerequisites that you have completed and are familiar with before beginning VM and App installation.
```

Comment:
if you are talking about the UI option Apps then it goes in bold, otherwise not in caps.
```suggestion
This article assumes that you have the latest release version of TrueNAS SCALE installed on your system. The following steps comprise a list of configuration prerequisites that you have completed and are familiar with before beginning VM and app installations.
```